utilityFunctions.js

```diff
@@ -0,0 +1,95 @@
+// I declare these lines so they are global here so i dont have to declare them in every function.
+// The append part does not work anymore because when reading the file user.json when not available the app crashes, i dont have a solution for this yet.
+
+const fs = require('fs');
+const fileData = JSON.parse(fs.readFileSync('user.json'));
+const path = './user.json';
+
+const validationUserPost = (userPostData) => {
+    const postedName = userPostData.name;
+    const postedHair = userPostData.hair;
+    const postedHeight = userPostData.height;
+    var regEx = /^[A-Za-z]+$/;
+    if (postedName.match(regEx) && postedHair.match(regEx) &&
+        postedName.length >= 4 && postedName.length <= 50 && postedHeight > 50 && postedHeight < 250) {
+        return true;
+    } else {
+        return false;
+    }
+}
+
+const addUserPost = (userPostData) => {
+    if (fs.existsSync(path) != true) {
+        fs.appendFileSync('user.json', JSON.stringify([userPostData], null, 2));
+        return false;
+    } else {
+        fileData.push(userPostData)
+        fs.writeFileSync('user.json', JSON.stringify(fileData, null, 2));
+        return true;
+    }
+}
+
+const putUserbyId = (userPutData, parameterId) => {
+    if (fs.existsSync(path) != true) {
```

Comment:
Shouldn't you validate if the newer user contents are valid?

Reply:
this is the validation if the user.json file exists, the user validation for the put is now in the server file as mentioned in the previous comment.